internal/transport/http2_client.go

```diff
@@ -1265,7 +1265,11 @@ func (t *http2Client) handleGoAway(f *http2.GoAwayFrame) {
 		return
 	}
 	if f.ErrCode == http2.ErrCodeEnhanceYourCalm {
-		if logger.V(logLevel) {
+		// When a client receives a GOAWAY with error code ENHANCE_YOUR_CALM and debug
+		// data equal to ASCII "too_many_pings", it should log the occurrence at a log level that is
+		// enabled by default and double the configure KEEPALIVE_TIME used for new connections
+		// on that channel.
+		if string(f.DebugData()) == "too_many_pings" {
 			logger.Infof("Client received GoAway with http2.ErrCodeEnhanceYourCalm.")
```

Comment:
Switch to default log level, not Info, and also perhaps add the doubling of keep alive time here?

Reply:
Btw, if you switch this to error, some tests might fail, as the test runner fails on seeing any unexpected error logs.

Reply:
Got it, thanks